src/components/board/board.test.jsx

```diff
@@ -1,19 +1,21 @@
 import React from 'react';
-import Board from './board';
 import { shallow, mount } from 'enzyme';
+import Board from './board';
 
-it('<Board/> renders without error', () => {
-  let squares = Array(9).fill(null);
-  shallow(<Board squares={squares} />);
-});
+describe('Board', () => {
```

Comment:
A small convention that we follow at GK in our describe for components is using the following syntax:
```js
describe('<Board />', () => {
```